src/screens/Analytics/GlobalSearchResults/GlobalSearchTables/Refunds/RefundsTableEntity.res

```diff
@@ -211,7 +211,15 @@ let getHeading = colType => {
 let getCell = (refundsObj, colType): Table.cell => {
   switch colType {
   | InternalReferenceId => Text(refundsObj.internal_reference_id)
-  | RefundId => Text(refundsObj.refund_id)
+  | RefundId =>
+    CustomCell(
+      <HSwitchOrderUtils.CopyLinkTableCell
+        url={`/refunds/${refundsObj.refund_id}`}
+        displayValue={refundsObj.refund_id}
+        copyValue={refundsObj.refund_id->Some}
```

Comment:
```suggestion
        copyValue={Some(refundsObj.refund_id)}
```

Can we do in this way as it state better readability

Reply:
https://github.com/juspay/hyperswitch-control-center/pull/1082#discussion_r1700529485